app/models/manageiq/providers/openstack/manager_mixin.rb

```diff
@@ -141,7 +141,7 @@ def translate_exception(err)
   end
 
   def verify_api_credentials(options = {})
-    options[:service] = "Compute"
+    options[:service] = "Identity"
```

Comment:
https://github.com/ManageIQ/manageiq-providers-openstack/pull/62 actually makes this PR unnecessary.